README.md

```diff
@@ -8,7 +8,17 @@ Open the command pallet and type `>Lua Format` to run the formatter.
 
 ## Requirements
 
-Formatter only works with lua version 5.1.
+Formatter only works with Lua version 5.1. In addition, the complete path to Lua must be specified in the configs such as:
+```
+Mac
+vscode-metalua-formatter.luaPath = '/Users/<user>/.lenv/lua/5.1.5/bin/lua'
+
+Linux
+vscode-metalua-formatter.luaPath = '/home/<user>/.lenv/lua/5.1.5/bin/lua'
+```
+You can find the complete path by running the following command:
+```which lua```
```

Comment:
Well maybe. Depends on which lua you are using. If you are using 5.3.5 like most people the number will be wrong.

Reply:
I specified that lua 5.1 is required for the formatter to work

Reply:
Yeah, but you are saying you can get the path for it using `which lua`, which may not be true depending how `lenv` was last configured. You may want to expand it to:

```
lenv install 5.1.5
lenv use 5.1.5
which lua
```

or just remove the line all together.